.goreleaser.yaml

```diff
@@ -0,0 +1,201 @@
+project_name: sbom-convert
+before:
+  hooks:
+    - go mod tidy
+    - ./scripts/completions.sh {{ .ProjectName }}
+    - ./scripts/manpages.sh {{ .ProjectName }}
+    - go generate ./...
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - "386"
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - "7"
+    ignore:
+      - goos: windows
+        goarch: arm
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w -X {{ .ModulePath }}/cmd/cli.version={{.Version}}
```

Comment:
Awesome, this should allow the CLI to run `go-cli --version` like before or do we have some other way to get the version from the CLI?

Reply:
ldflags works the same way, so just `--version` (-v is for verbose so only --version)

![image](https://github.com/bom-squad/go-cli/assets/116580393/08de7421-14c5-41d4-ba7d-2a794663bfa6)